ibis/expr/types.py

```diff
@@ -648,11 +648,12 @@ def param(type, name=None):
     return expr.name(name)
 
 
-def distinct_roots(*args):
-    all_roots = []
-    for arg in args:
-        all_roots.extend(arg._root_tables())
-    return list(toolz.unique(all_roots, key=id))
+def distinct_roots(*expressions):
```

Comment:
isn't this very similar to ``physical_tables``?

Reply:
Yes except that `physical_tables` will flatten some selection expressions, this doesn't

Reply:
ok, maybe add a doc-string to that effect? also should these be next to each other?

Reply:
I'll merge this PR, then push a commit to master with this docstring change.